mc2/views.py

```diff
@@ -20,6 +21,14 @@
 class HomepageView(ControllerViewMixin, ListView):
     template_name = 'mc2/home.html'
 
+    def get_context_data(self, *args, **kwargs):
+        context = super(HomepageView, self).get_context_data(*args, **kwargs)
+        active_org = active_organization(self.request)
+        if active_org:
+            context.update(
+                {'is_admin': active_org.has_admin(self.request.user)})
+        return context
+
```

Comment:
What does `get_context_data` do? I'm assuming it's some kind of generic view magic.